google-cloud-spanner-hibernate-testing/src/test/java/org/hibernate/jpa/test/BaseEntityManagerFunctionalTestCase.java

```diff
@@ -62,20 +62,27 @@ protected StandardServiceRegistryImpl serviceRegistry() {
 		return serviceRegistry;
 	}
 
-	@Before
-	@SuppressWarnings( {"UnusedDeclaration"})
-	public void buildEntityManagerFactory() {
-		log.trace( "Building EntityManagerFactory" );
+	private static boolean isInitialized;
 
-		entityManagerFactory =  Bootstrap.getEntityManagerFactoryBuilder(
-				buildPersistenceUnitDescriptor(),
-				buildSettings()
-		).build().unwrap( SessionFactoryImplementor.class );
+	@AfterClass
+	@SuppressWarnings( {"UnusedDeclaration"})
+	public static void releaseResources() {
```

Comment:
What are you trying to do here?

Reply:
This one here is the clean up so I changed it from After to AfterClass, but the AfterClass requires the method to be static. Before it would create and delete tables for each test, but since none of the tests actually change the data or tables we can do it all together at the end.